packages/react-core/src/components/Alert/Alert.tsx

```diff
@@ -9,6 +9,13 @@ import { AlertContext } from './AlertContext';
 import maxLines from '@patternfly/react-tokens/dist/js/c_alert__title_max_lines';
 import { Tooltip } from '../Tooltip';
 
+export interface SVGIconProps extends Omit<React.HTMLProps<SVGElement>, 'size' | 'ref'> {
+  color?: string;
+  size?: 'sm' | 'md' | 'lg' | 'xl';
+  title?: string;
+  noVerticalAlign?: boolean;
```

Comment:
@boaz0 what does this block do?

Reply:
I think there is a problem with the linter. I hardcoded the SVGIconProps from `react-icons` to make sure the value passed to the `customIcon` is an icon and not just anything else. The problem is the linter tells me that I cannot export SVGIconProps in that way and I need to change it to something else. However, running `yarn lint:ts --fix` fixes the lint error but the build fails because the file it imports SVGIconProps does not exist.

I hardcoded it to verify that this is the problem. I will probably have to fix the linter before merging this because it's stupid to have this duplication.

Thanks for having a look :laughing:

Reply:
The linting rule sees that the import is in braces `import { SVGIconProps }` and thinks you're trying to import from a barrel file which is why it's throwing the error. You can add an ignore comment since it's a bug with the rule
```
// eslint-disable-next-line patternfly-react/import-tokens-icons
import { SVGIconProps } from '@patternfly/react-icons/dist/js/createIcon';
```

Reply:
I did that but the linter insists on updating it to `'@patternfly/react-icons/dist/js/icons/s-v-g-icon-props';`.
I had to tweak the linter a little bit to treat this particular source path to what you suggested (here: https://github.com/patternfly/patternfly-react/pull/5163)

Reply:
Thanks @boaz0 , that PR is merged now so you can rebase this PR :)